### ISSUE TITLE
No Speaker Notes

### ISSUE THREAD

**Issue by deadlytackler5** [yjwen/org-reveal#137]
First and foremost thanks for making my emacs experience awesome through your tool.
My org-version is 8.3.1 (20150831)
Somehow I am not able to view speaker notes despite of inserting this in my org file -
```
 * MNIT Jaipur, process on for inclusion in tender.
#+BEGIN_NOTES
Quote given
Samples Submitted through Jaipur Team
Green Building Certificate provided to Project Director
#+END_NOTES
```
part of html ,which I think refer to above, (am no expert of html/css) is -

```
<section id="slide-orgheadline2">
<h4 id="orgheadline2">C P W D</h4>
<ul>
<li class="fragment appear">Request made for presentation at Nirman Bhawan.</li>
<li class="fragment appear">MNIT Jaipur, process on for inclusion in tender.</li>

</ul>
<aside class="notes">
<p>
Quote given
Samples Submitted through Jaipur Team
Green Building Certificate provided to Project Director
</p>

</aside>
</section>
```
I tried 
``` #+REVEAL_PLUGINS: (highlight markdown notes) ```
and
``` #+reveal_plugins: notes ```

looked for some custom option which may disable the exportation of speaker notes as suggested by you in #issue45, however was unable to find anything similar to ``` setq comment-start "#" ``` in my init file.
Considering that it may be a problem in my end, How you suggest to go ahead to find a solution to this and activate speaker notes. By the way am using Chrome browser...

I'll appreciate your effort in hand-holding though it's too much for asking...

**Comment by markjfisher** [comment]
If you're not using a single file option (which the docs say doesn't work with speaker notes) then I think I have just found out why this is happening, as I'm seeing the same issue and am about to raise a bug.

Can you check your generated output contains the external javascript references.
You should be getting the "dependencies: [ ..." in the output html, and this should reference the notes.js from the root path of your reveal.js installation. If not, I'll link my bug when I raise it shortly...

**Comment by yjwen** [comment]
Yes. The bug has been fixed by the merging from #141. Could you please try again with the latest code?
